content/guides/getting-started.md

```diff
@@ -246,7 +246,7 @@ Version: webpack 2.2.0
 Time: 390ms
     Asset    Size  Chunks                    Chunk Names
 bundle.js  544 kB       0  [emitted]  [big]  main
-   [0] ./~/lodash/lodash.js 540 kB {0} [built]
+   [0] ./src/index.js 219 bytes {0} [built]
    [1] (webpack)/buildin/global.js 509 bytes {0} [built]
    [2] (webpack)/buildin/module.js 517 bytes {0} [built]
    [3] ./src/index.js 278 bytes {0} [built]
```

Comment:
I am using webpack 3.0.0. I assumed that the output log will always start with the entry file. It is a logical assumption; isn't it? If that is not the case or it can't be guaranteed, a generic guide line as you have mentioned will do.